### DISCUSSION TITLE
Express LRS Release 2.5.0 channel 5 issue

### DISCUSSION THREAD

**Discussion by seemeeu** [ExpressLRS/ExpressLRS#1639] [General]
## Current Behavior
Express LRS Release 2.5.0 channel 5 issue
This channel is a bit value not the full range. 

Using the MATEK CRSF-SBUS ELRS CRSF TO SBUS CONVERTER on Flight Condolers ZOHD  kopilot lite and efei SN Sparrow V2 are locked down to use channel 5 as mode switch - Low (1000), middle (1500) and high (2000)

## Steps to Reproduce
Tested multiple Express LRS receivers 
Jumper ELRS Aion Rx Mini 2.4GHZ 16CH Receiver 
HappyModel ExpressLRS ELRS 2.4G EP1 and EP2

Tested multiple TX modules
BETAFPV ExpressLRS ELRS Micro RF TX Module 
HappyModel ES24TX Pro 1000mW

**Comment by pkendall64** [comment]
So you should connect to the WebUI on the RX and map another channel from the handset to the Ch5 output pin.

This is not a bug, but rather a question which would be best answered on the discord server in the #help-and-support channel or in one of the discussion threads. I will convert this to a discussion.
